### DISCUSSION TITLE
"Update all"

### DISCUSSION THREAD

**Discussion by peter9811** [NexovaDev/UpdateHub#229] [Ideas]
Anyway to add "Update all" button at the end?

**Comment by onyx-nxt** [comment]
We've received a lot of messages requesting this feature, so we've decided to add it in future updates. But first, we want to clear all potential bugs with the app install flow and then start working on an Update All button. Will keep you updated on when we add it

**Comment by peter9811** [comment]
Sounds great. The only big I found is, that sometimes don't show the "process" to the downloading file
The rest, everything is fine

**Reply by onyx-nxt** [reply]
Do you mean the progress bar?
